python/tvm/relay/frontend/tflite.py

```diff
@@ -846,7 +846,7 @@ def convert_shape(self, op):
         input_tensors = self.get_input_tensors(op)
         assert len(input_tensors) == 1, "input tensors length should be 1"
 
-        out = _op.shape_of(self.get_tensor_expr(input_tensors[0]))
+        out = shape_of(self.get_tensor_expr(input_tensors[0]))
```

Comment:
Before this PR, tflite mobilenet was returning a dynamic shape output :man_facepalming: